### ISSUE TITLE
Display ellipsis when table could not be fully displayed

### ISSUE THREAD

**Issue by pslacerda** [BastiaanBergman/tabel#4]
My table don't fit entirely on the screen, it seems truncated.

**Comment by BastiaanBergman** [comment]
rows will truncate when represented on the command line, `Tabel.max_repr_rows` is what can be adjusted to whatever number of rows you'd like to see. This behavior is similar to numpy and pandas, where very large arrays/DataFrames are summarized when displayed.

All data is still there, just not printed to the screen.

**Comment by pslacerda** [comment]
When `display`ed on JupyterLab It shows the 0 axis truncated without ellipsis and the 1 axis overflowed.

If I remember correctly, on the terminal it show the 0 axis truncated without ellipsis.

Anyway seems that tabulate don't have an output style like pandas or numpy to pretty print on JupyterLab.

**Comment by BastiaanBergman** [comment]
Correct, there is no ellipsis when truncating the 0 axis and the 1 axis will overflow when too big. There is also no html-styled pretty print, just plain ascii. On the plus side, there is a summary row noting how many rows there are (from which one can get the hint that not everything is displayed) and the whole package is lightweight and simple. I'm not ready to open the display-it-nicer-with-html can of worms but maybe one day we will.
